monitor-function-calls/package.json

```diff
@@ -18,7 +18,7 @@
   },
   "dependencies": {
     "bignumber.js": "^9.0.2",
-    "forta-agent": "^0.0.33"
+    "forta-agent": "^0.0.41"
```

Comment:
is there a reason for keeping this 0.0.41 (vs 0.1.1)?

Reply:
Just a mistake, good catch.  Thanks!